deltacat/tests/compute/test_compact_partition_incremental.py

```diff
@@ -325,24 +230,17 @@ def test_compact_partition_incremental(
     # execute
     rcf_file_s3_uri = compact_partition_func(compact_partition_params)
     # validate
-    _, rcf_object_key = rcf_file_s3_uri.rsplit("/", 1)
-    rcf_file_output: Dict[str, Any] = read_s3_contents(
-        setup_s3_resource, TEST_S3_RCF_BUCKET_NAME, rcf_object_key
+    compacted_delta_locator = get_compacted_delta_locator_from_rcf(
+        setup_s3_resource, rcf_file_s3_uri
     )
-    round_completion_info = RoundCompletionInfo(**rcf_file_output)
-    print(f"rcf_file_output: {json.dumps(rcf_file_output, indent=2)}")
-    compacted_delta_locator = round_completion_info.compacted_delta_locator
     tables = ds.download_delta(compacted_delta_locator, **ds_mock_kwargs)
     compacted_table = pa.concat_tables(tables)
     assert compacted_table.equals(
-        expected_result
-    ), f"{compacted_table} does not match {expected_result}"
+        expected_terminal_compact_partition_result
+    ), f"{compacted_table} does not match {expected_terminal_compact_partition_result}"
```

Comment:
combine_chunks() and sort both the tables before comparing equality. The compactor can create multiple files (which is especially true for v2).

Reply:
Done